Xamarin.Forms.Platform.Android/AppCompat/Platform.cs

```diff
@@ -165,16 +163,18 @@ void INavigation.RemovePage(Page page)
 			throw new InvalidOperationException("RemovePage is not supported globally on Android, please use a NavigationPage.");
 		}
 
-		SizeRequest IPlatform.GetNativeSize(VisualElement view, double widthConstraint, double heightConstraint)
+		public static SizeRequest GetNativeSize(VisualElement view, double widthConstraint, double heightConstraint)
 		{
 			Performance.Start(out string reference);
 
 			// FIXME: potential crash
 			IVisualElementRenderer visualElementRenderer = Android.Platform.GetRenderer(view);
 
+			var context = visualElementRenderer.View.Context;
```

Comment:
Why not `_context`?

Reply:
> `public static SizeRequest GetNativeSize`

Because the whole point is to make it static.